src/main/java/util/Convertor.java

```diff
@@ -0,0 +1,13 @@
+package util;
+
+import java.time.DayOfWeek;
+import java.util.List;
+
+public class Convertor {
+
+    private static List<String> days = List.of("일", "월", "화", "수", "목", "금", "토");
```

Comment:
```suggestion
    private static final List<String> days = List.of("일", "월", "화", "수", "목", "금", "토");
```
final로 선언해도 좋을 것 같아요.
인텔리제이에서 하이라이팅, 밑줄 등 표시되는 것은 한번씩 체크해보면 좋을 것 같아요. 이유와 함께 제안해줘도 알려줘서 유용하답니다.
<img width="319" alt="스크린샷 2025-02-22 오후 6 13 08" src="https://github.com/user-attachments/assets/502ac99d-2010-4db0-a6ca-fc2360bfad88" />